content/chainguard/libraries/overview.md

```diff
@@ -59,8 +59,12 @@ contents into our production environment.
 ## Introduction 
 
 Chainguard Libraries builds all available libraries from source code in the
-secure Chainguard infrastructure and makes them available for you. It removes
-all the software supply chain problems for libraries:
+Chainguard Factory and makes them available for you. Chainguard Factory is a
```

Comment:
This is a small start for Chainguard Factory. Wording is from the press release and website stuff around Assemble.